docs/_posts/murat-gunay/2021-09-09-bert_token_classifier_ner_btc_en.md

```diff
@@ -69,7 +68,7 @@ val ner_converter = NerConverter()
 
 val pipeline =  new Pipeline().setStages(Array(documentAssembler, tokenizer, tokenClassifier, ner_converter))
 
-val data = Seq("Wengers big mistakes is not being ruthless enough with bad players.", "my dream FUUUUUULHAAAAAAM !!!..").toDF("text")
+val data = Seq("Wengers big mistakes is not being ruthless enough with bad players.", "my dream Fulham F.C. !!!..").toDF("text")
```

Comment:
Can you remove second example and use this one?

![image](https://user-images.githubusercontent.com/36634572/144835637-5e84b6af-026f-4351-b6ca-f749b087ee9f.png)